### DISCUSSION TITLE
how to disable user warning log

### DISCUSSION THREAD

**Discussion by HongsCode** [pydantic/pydantic#7276] [Question] [answered]
update pydantic 2.3.0version from 1.4.x

in pydantic 2.3 version, 
show warning log, using variable name 'model_'

but can't change to variable name, because we used to 'model_' database column name. 

I wonder that can disable user wanring log or anythin else?...

warning log

UserWarning: Field "model_name" has conflict with protected namespace "model_".
You may be able to resolve this warning by setting `model_config['protected_namespaces'] = ()`.

my code
`data = {
    'model_name': "ab",
}
class Device(BaseModel):
    model_name: Optional[str] = Field(None)

m = Device(**data)
print(m)`

**Comment by harunyasar** [comment, marked as answer]
You are right. This is what you could do. It is mentioned [here](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.protected_namespaces).

**Reply by HongsCode** [reply]
thank you
I had already seen this documentation, but I was using it incorrectly

**Reply by GiorgioSgl** [reply]
This is the correct link https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.protected_namespaces

**Comment by peter65374** [comment]
In AI and deep learning engineer zone, 'model_' (e.g. model_id, model_name, model_type, etc) are widely used as while pydantic is very popular in AI app...

**Comment by umaxfun** [comment]
And is it safe to just turn off the warning?
